### PR TITLE
fix highlighting behavior after undo/delete

### PR DESCRIPTION
**Pull request by AlexExcellence**
- currently when deleting inline highlights or undoing inline highlights, the changes are not shown in the code editor -> fixed
- there was another bug where the selection of the finger tool didn't disappear when dismissing -> fixed

### CODE REVIEW

**Review by crustypluto19 — APPROVED** [review]
Code looks good, undo works just fine now!

File: CodeEditor/Sources/CodeEditor/UXCodeTextViewRepresentable.swift [hunk 1]
```diff
@@ -202,31 +202,38 @@ public struct UXCodeTextViewRepresentable: UXViewRepresentable {
...
             textView.setContentOffset(CGPoint(x: 0, y: 0), animated: false)
             if let textStorage = textView.codeTextStorage {
-                textStorage.replaceCharacters(in: NSRange(location: 0, length: textStorage.length),
-                                              with: editorBindings.source.wrappedValue)
+                textStorage.replaceCharacters(
+                    in: NSRange(location: 0, length: textStorage.length),
+                    with: editorBindings.source.wrappedValue)
             } else {
                 assertionFailure("no text storage?")
+                print("test")
```
Comment (huber-florian) [inline CodeEditor/Sources/CodeEditor/UXCodeTextViewRepresentable.swift]: Do we really need this?

Comment (AlexExcellence) [reply CodeEditor/Sources/CodeEditor/UXCodeTextViewRepresentable.swift]: Oops, doesn't get called anyway but will remove it

**Review by tcw0 — APPROVED** [review]
Looks fine!

**Pull request merged**